infra/src/Pos/Infra/Communication/Types/Relay.hs

```diff
@@ -65,6 +66,10 @@ data DataMsg contents = DataMsg
     { dmContents :: !contents
     } deriving (Generic, Show, Eq)
 
+instance Bi (DataMsg TxMsgContents) where
```

Comment:
Shouldn't this be in `txp/src/Pos/Txp/Network/Types.hs` with `TxMsgContents` as per: https://iohk.myjetbrains.com/youtrack/issue/CDEC-401
@erikd

Reply:
Normally probably yes, and in the end, maybe yes, but right now, definitely no :).

What we are aiming for at the moment is to fix `txp` so it no longer depends in `infra` and in the end, make `infra` depend on `txp`. We also want to avoid doing this in one huge PR that takes a week or more to get ready and then days to review etc.

So, we are temporarily going to contravene our normal good practices and do what ever it takes to fix the way `txp` and `infra` depend on each other.